pkg/reconciler/pipelinerun/resources/conditionresolution.go

```diff
@@ -96,7 +97,9 @@ func (rcc *ResolvedConditionCheck) ConditionToTaskSpec() (*v1alpha1.TaskSpec, er
 	}
 
 	t := &v1alpha1.TaskSpec{
-		Steps: []v1alpha1.Step{rcc.Condition.Spec.Check},
+		TaskSpec: v1alpha2.TaskSpec{
```

Comment:
nit: in a bunch of other places we're not nesting this on a newline.  We're doing `t := &v1alpha1.TaskSpec{TaskSpec: v1alpha2.TaskSpec{`)

I really don't mind either way which format is used but figure we should probably be consistent.

Reply:
Usually I am doing `&v1alpha1.TaskSpec{TaskSpec: v1alpha2.TaskSpec{` if `TaskSpec` is the only field in the struct, otherwise I aligne them :sweat_smile: